kge/model/embedder/lookup_embedder.py

```diff
@@ -77,6 +78,49 @@ def normalize_embeddings(job):
 
             job.pre_batch_hooks.append(normalize_embeddings)
 
+    @torch.no_grad()
+    def init_pretrained(self, pretrained_embedder: "LookupEmbedder") -> None:
```

Comment:
the __init__ method should also have init_for_load_only. For example, when this is set, none of the initialization functions needs to be called.